frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -74,6 +79,26 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
   const usedCapacity = humanizeBinaryBytes(usedCapacityMetric);
   const totalCapacity = humanizeBinaryBytes(totalCapacityMetric);
 
+  const validateSC = React.useCallback(() => {
+    if (!selectedSCName) return t('ceph-storage-plugin~No StorageClass selected');
+    if (!isNoProvionerSC) return '';
+    if (isArbiterEnabled) {
+      return !isArbiterSC(selectedSCName, pvData, nodesData)
+        ? t(
+            'ceph-storage-plugin~The Arbiter stretch cluster requires a minimum of 4 nodes (2 different zones, 2 nodes per zone). Please choose a different storage class or create a new volume set that matches the minimum node requirement.',
+          )
```

Comment:
```suggestion
    if (isArbiterEnabled && !isArbiterSC(selectedSCName, pvData, nodesData) 
        return t(
            'ceph-storage-plugin~The Arbiter stretch cluster requires a minimum of 4 nodes (2 different zones, 2 nodes per zone). Please choose a different storage class or create a new volume set that matches the minimum node requirement.',
          )
```

Reply:
will be buggy this way... if `isArbiterEnabled === true && !isArbiterSC === false` then control will move to L92 and here `!hasFlexibleScaling === true && !isIntenalAttachedSC === true` (beacuse current storage-class has nodes w.r.t to arbiter configuration)... so it will start showing the incorrect error: "The OCS Storage cluster requires a minimum of 3 node....." even though my nodes/pvs configurations are correct w.r.t arbiter mode...

Reply:
We dont want it to be getting true for `isInternalAttachedSC`. Its only required to be true/false or checked when `failureDomain === 'rack' .